### DISCUSSION TITLE
Question: Is the NumberBox's Spin Button Disabled state correct?

### DISCUSSION THREAD

**Discussion by mdtauk** [microsoft/microsoft-ui-xaml#7466] [Q&A]
Looking at the NumberBox test page in the MUXTestApp - When the compact mode spin buttons are disabled, there is a backplate visible for both the up and down buttons.

It looks a little odd, so I was wondering if this is actually the correct design for these buttons?

![image](https://user-images.githubusercontent.com/7389110/122625964-bcbfdc80-d09f-11eb-9a44-eafceea7498c.png)
_Both buttons disabled_

![image](https://user-images.githubusercontent.com/7389110/122626008-f395f280-d09f-11eb-88d3-7422c7b06046.png)
_Top button enabled, bottom button hovered_

![image](https://user-images.githubusercontent.com/7389110/122626037-0e686700-d0a0-11eb-904e-168d95b4d513.png)
_Top button pressed, bottom button enabled_

By way of a comparison, here are the Inline spin buttons

![image](https://user-images.githubusercontent.com/7389110/122626075-466faa00-d0a0-11eb-8884-49e4f2a799db.png)
_both buttons enabled_

![image](https://user-images.githubusercontent.com/7389110/122626089-5ab3a700-d0a0-11eb-8bba-4c3a636233d8.png)
_both buttons disabled (these also show a backplate when disabled_

**Comment by StephenLPeters** [comment]
@chigy and @ranjeshj FYI

**Comment by StephenLPeters** [comment]
@teaP FYI
